src/components/AttachmentPicker/checkCameraPermission/index.ios.js

```diff
@@ -0,0 +1,8 @@
+import {PERMISSIONS, request, RESULTS} from 'react-native-permissions';
+
+export default function checkCameraPermission() {
```

Comment:
@dharmik, I was checking the code by pulling your branch and I noticed that we already have `launchCamera.js` can we just move this to `launchCamera.ios.js` and we won't need `checkCameraPermission` files at all.

Reply:
If we write that checkCameraPermission there so it will always false because we check permission before opening. For android we have permission as below which [Android API](https://reactnative.dev/docs/permissionsandroid) is available in React Native.

`PermissionsAndroid.request(PermissionsAndroid.PERMISSIONS.CAMERA)`

Reply:
Okay